pkg/rulesConfig/defaultRules/10-ensure-sca-scanner.go

```diff
@@ -74,7 +76,7 @@ func githubErrorsRule10(githubData map[string]*githubConnector.GithubOwner) ([]*
 							}
 						}
 
-						for _, regexExpression := range runRegexExpressions {
+						for _, regexExpression := range sharedRegexExpressions {
```

Comment:
Just a side note, not necessarily have to fix - we could concatenate all the regex expressions using `|` and then do just one regex matching. It would be more efficient and would avoid the `for` loop.

Reply:
yeah I think more code wins the battle versus more complex regex